src/lib/pages/faucet/index.tsx

```diff
@@ -182,7 +194,7 @@ const Faucet = () => {
         isLoading={isLoading}
         disabled={disabled}
       >
-        Request 10 testnet OSMO
+        Request {faucetAmount} testnet {faucetDenom}
```

Comment:
Should we change the denominator to uppercase?

Reply:
Already turn it to uppercase on line 67